docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/yield-syntax.md

```diff
@@ -42,56 +45,57 @@ YIELD
 ```
 
 ```ngql
-nebula> GO FROM 201 OVER like YIELD like._dst AS id | YIELD $-.* WHERE $-.id == 200;
+nebula> GO FROM 100 OVER follow YIELD follow._dst AS id | YIELD $-.* WHERE $-.id == 106;
 
 =========
 | $-.id |
 =========
-| 200   |
+| 106   |
 ---------
 
-nebula> $var2 = GO FROM 200 OVER like;$var1 = GO FROM 201 OVER like; \
-    YIELD $var1.* UNION YIELD $var2.*;
-
-===================
-| $var1.like._dst |
-===================
-| 200             |
--------------------
-| 201             |
--------------------
-| 202             |
--------------------
+nebula> $var1 = GO FROM 101 OVER follow; $var2 = GO FROM 105 OVER follow; YIELD $var1.* UNION YIELD $var2.*;
+
+=====================
+| $var1.follow._dst |
+=====================
+| 100               |
+---------------------
+| 102               |
+---------------------
+| 104               |
+---------------------
+| 110               |
+---------------------
 ```
 
-### As Separate Statement
+### As Stand-alone Statement
 
-`YIELD` statement can be used independently to retrieve computation results without reference to any graph. You can use `AS` to rename an alias.
+`YIELD` statement can be used independently to retrieve computation results without reference to any graph. You can use `AS` to rename it an alias.
```

Comment:
You can use `AS` to rename it an alias. >- You can use `AS` to rename it as an alias.
This sentence is weird to me but I'm not sure if I fixed it.